circuits/ballotproof/circom2gnark.go

```diff
@@ -0,0 +1,39 @@
+package ballotproof
+
+import "github.com/vocdoni/circom2gnark/parser"
+
+// Circom2GnarkProof
+func Circom2GnarkProof(witness []byte) (*parser.GnarkRecursionProof, error) {
+	// create circom proof and public signals
+	circomProof, pubSignals, err := CompileAndGenerateProofForTest(witness)
```

Comment:
This function isn't supposed to be a test, so why this call?

Reply:
sorry but I don't understand the question :(

Reply:
Circom2GnarkProof() is supposed to be a function used by other packages when running in production mode. Why does it call "CompileAndGenerateProofForTest()" ?